scripts/generate-test-services.ts

```diff
@@ -35,7 +35,8 @@ const generatorConfigOData = {
   generateCSN: false,
   // Unnecessary options
   sdkAfterVersionScript: false,
-  s4hanaCloud: false
+  s4hanaCloud: false,
+  generateSdkMetadata: false
```

Comment:
minor: default is false.

Reply:
True but it is not optional so I made it optional as well.